packages/context/src/binding-decorator.ts

```diff
@@ -21,6 +21,25 @@ import {Constructor} from './value-promise';
  * Decorator factory for `@bind`
  */
 class BindDecoratorFactory extends ClassDecoratorFactory<BindingMetadata> {
+  mergeWithOwn(
```

Comment:
This change is needed to allow `bind` to be called multiple times on the same class, for example:

```ts
@booter('rest')
@extensionPoint(MODEL_API_BUILDER_PLUGINS)
export class RestBooter extends BaseArtifactBooter {
  // ...
}
```

Reply:
Fixed by https://github.com/strongloop/loopback-next/pull/3618